### PR TITLE
Admin Message Tab: Add Captcha column with checkbox #2796

### LINKED ISSUE #2796: Admin Message Tab: Add "Captcha" column with checkbox
In the admin message at https://susi.ai/admin/messages tab please add a column with the option to activate or deactivate captcha.

Screenshot from 2019-08-06 23-47-24

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2796

Changes: Admin Message Tab: Add Captcha column with a checkbox

Demo Link: https://pr-2867-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-08-19 at 1 29 54 AM

### CODE REVIEW

File: src/components/Admin/Messages/RecaptchaOptions.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        .catch(error => {
+          actions.openSnackBar({
+            snackBarMessage: 'Failed to change reCaptcha settings',
+          });
+        });
+    });
+  };
+
+  render() {
+    const { login, signUp, changePassword } = this.state;
```
Comment (akshatnitd) [inline src/components/Admin/Messages/RecaptchaOptions.js]: Are these still stored in state? Aren't they hardcoded?

File: src/components/Admin/Messages/RecaptchaOptions.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import Checkbox from '@material-ui/core/Checkbox';
+import { setRecaptchaConfig } from '../../../apis';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import uiActions from '../../../redux/actions/ui';
+import PropTypes from 'prop-types';
+
+class RecaptchaOptions extends React.Component {
+  state = {
+    login: false,
```
Comment (akshatnitd) [inline src/components/Admin/Messages/RecaptchaOptions.js]: These should be initialised with the store variables

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Comment by fragm3** [comment]
@akshatnitd made the changes.

**Comment by mariobehling** [comment]
At second thoughts, I think this could be confusing as we are following two different logics. Let's rather move this to https://pr-2867-fossasia-susi-web-chat.surge.sh/admin/settings please as a subtab.

**Comment by fragm3** [comment]
@mariobehling made the changes
Screenshot 2019-08-21 at 10 11 43 PM
Screenshot 2019-08-21 at 10 11 51 PM

File: src/components/Admin/Messages/message-config.js [hunk 3]
```diff
@@ -140,6 +142,15 @@ const MESSAGE_CONFIG = [
     subject: 'Testing email through Admin',
     options: 'Mail',
   },
+  {
+    recipients: 'User',
```
Comment (akshatnitd) [inline src/components/Admin/Messages/message-config.js]: We don't need this now.

File: src/components/Admin/Messages/RecaptchaOptions.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const RecaptchaOptions = ({ captchaConfig }) => {
+  const { login, signUp, changePassword } = captchaConfig;
+  return (
```
Comment (akshatnitd) [inline src/components/Admin/Messages/RecaptchaOptions.js]: We don't need this now

**Comment by fragm3** [comment]
@akshatnitd I am showing on-screen message as well, see: https://pr-2867-fossasia-susi-web-chat.surge.sh/admin/messages

**Pull request merged**